src/main/webapp/assets/js/config-swagger.js

```diff
@@ -0,0 +1,12 @@
+require.config({
```

Comment:
Somehow related, https://github.com/apache/brooklyn-ui/pull/38 activates the rat plugin for the project. So good point about adding license here.